src/realm.js

```diff
@@ -849,10 +849,10 @@ export class Realm {
     if (key === "") {
       // throw type error?
     }
-    this._abstractValuesDefined[key] = abstractValue;
+    this._abstractValuesDefined.set("key", abstractValue);
```

Comment:
typo, you have `"key"` rather than `key`.

Reply:
Hey, thanks.

Just realised tests would let me push just silly patches. Can you help me there? Refer [comment](https://github.com/facebook/prepack/pull/1197#issuecomment-347505927)